nx/source/services/usb.c

```diff
@@ -265,7 +263,7 @@ static Result _usbDsGetEvent(Service* srv, Handle* handle_out, u64 cmd_id) {
         rc = resp->result;
 
         if (R_SUCCEEDED(rc)) {
-            *handle_out = r.Handles[0];
+            eventLoadRemote(event_out, r.Handles[0], false);
```

Comment:
autoclear should probably be 'true' here, and instances of eventClear should be removed.

Reply:
I disagree, actually -- I think that autoclear should be false. The usb:ds API should provide a sufficiently low-level wrapping that custom interfaces can be written -- and so allowing manual clear should be the case.

Perhaps "eventSetAutoclear(Event *event, bool autoclear);" could be added, incidentally?

Reply:
The autoclear stuff parallels what Nintendo is doing in their IPC marshalling code. It seems like they decided some events will autoclear on the user's side, and some will never be cleared at all by the user. It sounds to me like this is one of the events that Nintendo mandated user-autoclear for.

Reply:
@fincs Correct, the relevant SystemEvents pass AutoClear = false for usb.

Reply:
Fair enough. Seems like a weird design choice though.